core/chains/evm/txmgr/eth_broadcaster.go

```diff
@@ -375,47 +375,46 @@ func (eb *EthBroadcaster) handleInProgressEthTx(ctx context.Context, etx EthTx,
 		return errors.Wrap(err, "building transmit checker")
 	}
 
+	lgr, err := etx.GetLogger(eb.logger.With(
+		"gasPrice", attempt.GasPrice,
+		"gasTipCap", attempt.GasTipCap,
+		"gasFeeCap", attempt.GasFeeCap,
+	))
+	if err != nil {
+		eb.logger.Errorw("failed to get tx logger meta of the transaction", "err", err)
+	}
+
 	// If the transmit check does not complete within the timeout, the transaction will be sent
 	// anyway.
 	checkCtx, cancel := context.WithTimeout(ctx, TransmitCheckTimeout)
 	defer cancel()
-	err = checker.Check(checkCtx, eb.logger, etx, attempt)
+	err = checker.Check(checkCtx, lgr, etx, attempt)
 	if errors.Is(err, context.Canceled) {
-		eb.logger.Errorw("Transmission checker timed out, sending anyway",
-			"ethTxId", etx.ID,
-			"meta", etx.Meta,
-			"checker", etx.TransmitChecker)
+		lgr.Error("Transmission checker timed out, sending anyway")
 	} else if err != nil {
 		etx.Error = null.StringFrom(err.Error())
-		eb.logger.Infow("Transmission checker failed, fatally erroring transaction.",
-			"ethTxId", etx.ID,
-			"meta", etx.Meta,
-			"checker", etx.TransmitChecker,
-			"err", err)
-		return eb.saveFatallyErroredTransaction(&etx)
+		lgr.Infow("Transmission checker failed, fatally erroring transaction.", "err", err)
```

Comment:
This seems to need to be bumped to warn imo based on [logging guidelines](https://github.com/smartcontractkit/chainlink/blob/develop/core/logger/logger.go), I think I missed this one in my audit